spec/apibuilder-spec.json

```diff
@@ -17,282 +17,293 @@
   "enums": {
     "method": {
       "values": [
-	{ "name": "GET" },
-	{ "name": "POST" },
-	{ "name": "PUT" },
-	{ "name": "PATCH" },
-	{ "name": "DELETE" },
-	{ "name": "HEAD" },
-	{ "name": "CONNECT" },
-	{ "name": "OPTIONS" },
-	{ "name": "TRACE" }
+        { "name": "GET" },
```

Comment:
Looks like indentation got messed up in this file, generating a lot of noise.

Reply:
The document was originally indented with a mix of tab stops and spaces. It is now indented with just spaces.

Reply:
Prob better to do such extensive indentation reformatting in a separate PR.